index.html

```diff
@@ -226,6 +231,15 @@ <h2>I am a Software Developer</h2>
             </ul>
           </div>
         </div>
+        <div class="desktop-headline-bg-section d-flex">
```

Comment:
Please make the icons clickable links. KIndly see the screenshot below for reference
![icons](https://user-images.githubusercontent.com/35504149/135511222-6c46f2bf-57ef-44b0-83c8-0aac8fd3ccd8.png)
e.